### ISSUE TITLE
Shadow fiend and Dota+

### ISSUE THREAD

**Issue by UZURATI0N** [jeffhill/Dota2#1609]
I ran into a problem when completing Dota + tasks on the hero Shadow Fiend, I fully fulfill the condition of the task, but this does not count. Tried to do it in turbo and all pick.How to deal with this problem?

**Comment by LokeLok** [comment]
Did you win the game you completed the quest in?

**Comment by UZURATI0N** [comment]
> Вы выиграли игру, в которой выполнили квест?

of course

**Comment by UZURATI0N** [comment]
> Вы должны сделать запись происходящего. Я никогда не сталкивался с этим и никогда не слышал, чтобы у кого-то была эта проблема.

in terms of what I should write down how I go through the tasks and they are not completed?

**Comment by LokeLok** [comment]
Yes, if there is a way to reproduce it or to document the bug, it's easier to understand whats going on.
